src/components/EmbeddedAskForm.scss

```diff
@@ -15,11 +18,16 @@
       }
     }
   }
+
+  header {
+    display: none !important;
+  }
```

Comment:
I don't think we can or should do this, it breaks the ability of the embedding organization to use Ask's built in form editor to specify this text, which I think will be unintuitive. I'm going to fix the merge conflict, then probably add a commit to omit this, so that we're using Ask as the source of truth for the intro copy, not the spreadsheet; the spreadsheet (or sidecar CSV or JSON) can be used for more editorial text relating to the vis itself.